### ISSUE TITLE
Statement of Assets can't filter by Taxonomy

### ISSUE THREAD

**Issue by metal450** [portfolio-performance/portfolio#2045]
On Statement of Assets, we are able to filter by accounts - but it would be nice to be able to filter by taxonomies too.

**Comment by flywire** [comment]
https://forum.portfolio-performance.info/t/performance-by-taxonomy/17232

Add group by Taxonomy in reports too.

**Comment by Drkmstr310** [comment]
Yes, this would be very helpful. Anyone with knowledge of the code might be able to help? :/
